### DISCUSSION TITLE
power bar

### DISCUSSION THREAD

**Discussion by Davdadev** [rocksdanister/lively#2204] [General]
Create a green loading bar  the bottom of your screen that works with translucentTB that indicates your charge level

**Comment by rocksdanister** [comment]
Should be pretty straightforward creating a html page with progressbar:
https://www.w3schools.com/tags/tag_progress.asp

**Comment by Davdadev** [comment]
for your desk top

**Reply by rocksdanister** [reply]
Lively supports webpage wallpapers, all the included wallpapers are using html.
